### PR TITLE
fix(frontend): add help text to story form

### LINKED ISSUE #2364: Success Stories submission form - Add help text requesting Markdown formatting to the Content box
**Is your feature request related to a problem? Please describe.**
When users use [the Success Stories submission form](https://www.python.org/success-stories/create/) to submit a success story, there is currently no guidance on whether to submit Success Stories in plain text or a mark up language. It's much faster on our end to post stories if they are submitted in Markdown so we don't have to format them ourselves. 

**Describe the solution you'd like**
I'd like for there to be help text next to the Content text entry box on the Success Stories form indicating Markdown is preferred.

**Describe alternatives you've considered**
Currently we either reach out to request the submitter put their submission into Markdown after we receive it or we put it in Markdwown ourselves.

**Additional context**
Suggested text to add to [the Success Stories submission form](https://www.python.org/success-stories/create/) is:
"Submissions in Markdown are strongly preferred and can be processed more quickly" as shown in the attached screenshot in red. 
(I'm not 100% sure that wording is the accurate way to describe text being "in Markdown"; please edit if it doesn't quite make sense as I've written it. Thank you!)
Screenshot of pythondotorg successs story form

@ewdurbin

### PR DESCRIPTION
**Pull request by JacobCoffee**
#### Description

- Adds help text to this content field. Initially I tried https://github.com/agusmakmun/django-markdown-editor so that users could have a nice WYSIWYG editor for Markdown and it would improve the UX but it was kind've not great. I probably just didn't set it up right though.

image

#### Closes

- Closes #2364

### CODE REVIEW

**Comment by ewdurbin** [comment]
Suggestion: Set the `help_texts` of the `Meta` class of the [form](https://github.com/python/pythondotorg/blob/main/successstories/forms.py#L12) to avoid generating a migration.

**Comment by ewdurbin** [comment]
See: https://docs.djangoproject.com/en/dev/topics/forms/modelforms/#overriding-the-default-fields

**Comment by JacobCoffee** [comment]
@ewdurbin neat! done in [c338929](https://github.com/python/pythondotorg/pull/2594/commits/c33892944351cc841afdb6cae4c4f31890eba904)

File: successstories/forms.py [hunk 1]
```diff
@@ -24,6 +24,9 @@ class Meta:
         labels = {
             'name': 'Story name',
         }
+        help_texts = {
+            "content": "Note: Submissions in Markdown are strongly preferred and can be processed faster.",
```
Comment (ewdurbin) [inline successstories/forms.py]:
```suggestion
            "content": "Note: Submissions in <a href="https://www.markdownguide.org/basic-syntax/">Markdown</a> are strongly preferred and can be processed faster.",
```

Or some other introductory resource?

**Review by ewdurbin — APPROVED** [review]
One idea, but not a blocker.

File: successstories/forms.py (hunk 1 above)
Comment (JacobCoffee) [reply successstories/forms.py]: I like that, considering target audience 

image

[cc85402](https://github.com/python/pythondotorg/pull/2594/commits/cc85402b3104dbc6892fb525c3d7ae9015d6600a)

**Pull request merged**